README.md

```diff
@@ -116,39 +116,41 @@ sub-directory and the resulting source-git repo is going to be stored in
 
 ## Tests
 
-**The test suite works only with root podman, so please make sure to run it
-with `CONTAINER_ENGINE='sudo podman'`.** (the reason is the way how UIDs are
-mapped when running rootless)
+The `tests_image` test suite aims to test the ability to convert
```

Comment:
How about making a note here that his needs to be run with `TEST_TARGET=tests_image make check`?

Also, it would be nice to explain here, what types of issues this test suite is expected to catch. Looking at the two test suites and the way they are run, I see very little difference, but I'm pretty sure I'm missing something important here.

Reply:
> How about making a note here that his needs to be run with `TEST_TARGET=tests_image make check`?

OK, what about a new make target?

> Also, it would be nice to explain here, what types of issues this test suite is expected to catch. Looking at the two test suites and the way they are run, I see very little difference, but I'm pretty sure I'm missing something important here.

The `tests` test suite:
- Runs directly on host => needs all dependencies locally
- Does not run any image => can be run inside a container
- Should contain all the tests and tests the functionality.
  - (I am adding there the tests for updates.)

The `tests_image`:
- pytest runs on host
- dist2src runs in a container => cannot be run inside container
- Should cover only basic test to check that we don't break provided image.
  - Uses only one example package.
- In README:
  - > The `tests_image` test suite aims to test the ability to convert
- In tests:
  - > Check only that the image can be used for conversion. Conversion process is tested in the regular test suite.

Reply:
The `tests_image` suite triggers `make run` with a `dist2src` command. `make run` runs this command in a container created from `IMAGE_NAME`.

`make check-in-container` runs the `tests` test suite in a container created from `IMAGE_NAME`. The tests execute `dist2src` commands to test the conversion.

To me it seems these two approaches check the same thing: it's possible to convert a repo running `dist2src` in the container. The only difference is where `pytest` is running (on the host or in the container), but I don't find this aspect relevant in telling whether the script/container image combo can be used for conversion.

Reply:
I can remove the old way, but I wanted to make sure that the `make run` works for the future ourselves.

With the "new" approach we don't check:
- that with `make run` the container is correctly created and run with the right options and correct mounting
- that the CLI works well and the package was installed correctly (we don't use it in `tests`)

The `tests_image` runs only two (single) tests as smoke/integration tests. Is it such a big problem to have them?

Reply:
The `tests_image` removed.